src/StreamJsonRpc/Reflection/RpcTargetInfo.cs

```diff
@@ -488,7 +493,8 @@ void ActOn([DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicMeth
     /// </summary>
     /// <param name="exposedMembersOnType">Type to reflect over and analyze its events.</param>
     /// <returns>A list of EventInfos found.</returns>
-    [SuppressMessage("Trimming", "IL2075:'this' argument does not satisfy 'DynamicallyAccessedMembersAttribute' in call to target method. The return value of the source method does not have matching annotations.", Justification = "false positive: https://github.com/dotnet/linker/issues/1731")]
+    [UnconditionalSuppressMessage("Trimming", "IL2065:UnrecognizedReflectionPattern", Justification = "false positive: https://github.com/dotnet/linker/issues/1731")]
+    [UnconditionalSuppressMessage("Trimming", "IL2075:UnrecognizedReflectionPattern", Justification = "false positive: https://github.com/dotnet/linker/issues/1731")]
```

Comment:
This could also be under ifdef (same limitation around recognizing foreach applies.)

The one later in this file too.